README.md

```diff
@@ -139,14 +146,24 @@ Describe to him what a RESTful route is, and list what the seven RESTful routes
 would look like for such an application.
```

Comment:
For question 6 did you have trouble the question phrasing or the subject matter?